From 8c8a0f53a1f90b1bca85eec8e21633c770d30c13 Mon Sep 17 00:00:00 2001
From: Kha Tran <kha.tran.px@rvc.renesas.com>
Date: Thu, 25 Jan 2018 09:33:35 +0700
Subject: [PATCH 422/504] ARM: dts r8a7744: Add vspd{0,1} and vspm nodes

Add the vspm, vspd0 and vspd1 which are used by DU to SoC dtsi
of R8A7744.

Signed-off-by: Kha Tran <kha.tran.px@rvc.renesas.com>
Signed-off-by: Loc Vu <loc.vu.zn@renesas.com>
Signed-off-by: Hao Bui <hao.bui.yg@renesas.com>
---
 arch/arm/boot/dts/r8a7744.dtsi | 24 ++++++++++++++++++++++++
 1 file changed, 24 insertions(+)

diff --git a/arch/arm/boot/dts/r8a7744.dtsi b/arch/arm/boot/dts/r8a7744.dtsi
index 9aaff17..5db9252 100644
--- a/arch/arm/boot/dts/r8a7744.dtsi
+++ b/arch/arm/boot/dts/r8a7744.dtsi
@@ -1636,6 +1636,30 @@
 			power-domains = <&sysc R8A7744_PD_ALWAYS_ON>;
 		};
 
+		vspm@fe928000 {
+			compatible = "renesas,vspm-vsps","renesas-vspm";
+			reg = <0 0xfe928000 0 0x7404>;
+			interrupts = <0 267 IRQ_TYPE_LEVEL_HIGH>;
+			clocks = <&mstp1_clks R8A7744_CLK_VSP1_S>;
+			power-domains = <&sysc R8A7744_PD_ALWAYS_ON>;
+		};
+
+		vspd0: vspd0@fe930000 {
+			compatible = "renesas,vsp2";
+			reg = <0 0xfe930000 0 0x7404>;
+			interrupts = <0 246 IRQ_TYPE_LEVEL_HIGH>;
+			clocks = <&mstp1_clks R8A7744_CLK_VSP1_DU0>;
+			power-domains = <&sysc R8A7744_PD_ALWAYS_ON>;
+		};
+
+		vspd1: vspd1@fe938000 {
+			compatible = "renesas,vsp2";
+			reg = <0 0xfe938000 0 0x7404>;
+			interrupts = <0 247 IRQ_TYPE_LEVEL_HIGH>;
+			clocks = <&mstp1_clks R8A7744_CLK_VSP1_DU1>;
+			power-domains = <&sysc R8A7744_PD_ALWAYS_ON>;
+		};
+
 		du: display@feb00000 {
 			compatible = "renesas,du-r8a7744";
 			reg = <0 0xfeb00000 0 0x40000>,
-- 
2.7.4

